website/docs/automation/gitops/argocd/app-of-apps/index.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Can you limit this page to an introduction to the concept of app of apps and move the hands-on part in to a sub-page called "setup" or something similar?

Reply:
I created "Setup" section